public/components/navBar.js

```diff
@@ -0,0 +1,18 @@
+fetch("/templates/navBar.html")
+    .then(stream => stream.text())
+    .then(text => label(text));
+
+function label(html) {
```

Comment:
change the function name to something more self-documenting... defineElement? defineNavBar?